.github/workflows/gccgo.yml

```diff
@@ -0,0 +1,15 @@
+on: [push, pull_request]
+jobs:
+  gccgo:
+    runs-on: ubuntu-latest
+    name: "gccgo Build"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install build utils
+        run: |
+          sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
+          sudo apt-get update
+          sudo apt-get install -y gccgo-11
+      - run: go-11 version
```

Comment:
Which Go version is this using? Can we specify the versions here, to make sure we're testing on Go 1.16 and Go 1.17?

Reply:
https://golang.org/doc/install/gccgo
"The GCC 11 releases include a complete implementation of the Go 1.16.3 release."

Reply:
Thank you. And it looks like there's no version for 1.17. I guess we'll have to live with that.